crates/re_types_builder/build.rs

```diff
@@ -70,12 +70,14 @@ fn main() {
 
     // NOTE: This requires `flatc` to be in $PATH, but only for contributors, not end users.
     // Even for contributors, `flatc` won't be needed unless they edit some of the .fbs files.
-    let sh = Shell::new().unwrap();
+    let sh = Shell::new().expect("Shell::new() failed");
+    #[allow(clippy::unwrap_used)] // unwrap is okay here
```

Comment:
someething went sideways here or is `cmd!` internally doing an unwrap?

Reply:
I have fixed hopefully all the linting errors.
> something went sideways here or is cmd! internally doing an unwrap?

I'm not sure I understand, does it fail somewhere? I haven't seen anything in the logs.

Reply:
Btw, locally I get another error when running `cargo test --quiet --all-targets`:
(I added some logs to  `bench_downcast_first` in `rerun/crates/re_data_store/benches/arrow2.rs`):

```
bench_downcast_first: re_arrow2::array::fixed_size_list::FixedSizeListArray for kind: Struct
bench_downcast_first: cell:DataCell { inner: DataCellInner { name: "example.MyPoint", size_bytes: 0, values: StructArray[{x: 0, y: 0}] } }
thread 'main' panicked at crates/re_data_store/benches/arrow2.rs:298:30:
called `Option::unwrap()` on a `None` value
```

I don't know this code well but it seems some dynamic casting goes haywire. 
The culprit seems to be this line:
`ArrayKind::Struct => bench_downcast_first::<FixedSizeListArray>(&mut group, kind)`
shouldn't it be?
`ArrayKind::Struct => bench_downcast_first::<StructArray>(&mut group, kind)`

...because judging from the logs above `generate_cells` generates a `StructArray` for `MyPoint`. 
`impl Loggable for MyPoint`also uses `Struct`/`StructArray` and not `FixedSizeListArray`.

I don't think this is caused by my changes though - and ideas?